src/EnergyPlus/WindowManager.cc

```diff
@@ -6070,7 +6070,7 @@ namespace WindowManager {
 			TInRadFr = TInRad * root_4( ( 1.0 + 0.5 * ProjCorrFrIn ) / ( 1.0 + ProjCorrFrIn ) );
 			FrameCon = SurfaceWindow( SurfNum ).FrameConductance;
 			HInRad = 0.5 * SurfaceWindow( SurfNum ).FrameEmis * sigma * pow_3( TInRadFr + SurfaceWindow( SurfNum ).FrameTempSurfIn + TKelvin );
-			HInConvFr = 0.0;
+			HInConvFr = HInConv;
```

Comment:
@EnergyArchmage Doesn't EnergyPlus remove the frame area from the base surface area even if there is no frame projection? It seems like a bad assumption that no projection implies no frame. Maybe I'm not understanding you correctly.